prymer/primer3/primer3_parameters.py

```diff
@@ -196,6 +209,9 @@ class ProbeParameters:
     probe_max_dinuc_bases: int = 4
     probe_max_polyX: int = 5
     probe_max_Ns: int = 0
+    probe_max_self_any_thermo: Optional[float] = None
+    probe_max_self_end_thermo: Optional[float] = None
+    probe_max_hairpin_thermo: Optional[float] = None
```

Comment:
@tfenne Same question as above - do you intend for these to be non-`Optional`, if we are going to enforce that `primer3` will always return the thermodynamic results?

Reply:
I think it is consistent if these params stay optional -- if they are not provided, then we default to `probe_tm.min - 10.0` in the `__post_init__`, which is what the Primer3 manual says it does.